src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
 
+    public static final int SUPPLY_INDEX = 0;
+    public static final int BUY_INDEX = 1;
+    public static final int RESULT_INDEX = 2;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String stringLine;
+        int[] values = {0, 0, 0};
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            while ((stringLine = br.readLine()) != null) {
+                if (stringLine.split(",")[0].equals("supply")) {
+                    values[SUPPLY_INDEX] += Integer.valueOf(stringLine.split(",")[1]);
+                }
+                if (stringLine.split(",")[0].equals("buy")) {
+                    values[BUY_INDEX] += Integer.valueOf(stringLine.split(",")[1]);
```

Comment:
create two more methods, one for reading from file and the second one for forming statistic